src/packages/sdk/src/__tests__/fixtures/getSchema/pkg-json-valid-relative-path/package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "pkg-json-valid-path",
+  "prisma": {
+    "schema": "./db/schema.prisma"
```

Comment:
Does this feature also work with _absolute_ paths?

Reply:
yes, i'm checking for that